broker/network/network.go

```diff
@@ -55,13 +55,19 @@ func NewServer(c inter.Framework, bi BuildInfoProvider, debug bool) (s *Server,
 		mu:      &sync.Mutex{},
 	}
 
+	// Validate timeouts
+	if s.config.Choria.NetworkAuthTimeout > 60*time.Second || s.config.Choria.NetworkTLSTimeout > 60*time.Second {
+		return nil, fmt.Errorf("NetworkAuthTimeout and NetworkTLSTimeout must not exceed 60 seconds")
```

Comment:
Elsewhere like here https://github.com/choria-io/go-choria/blob/a8957975c7dc9f232d41d92eb5f1eea4d4d55243/broker/network/network.go#L161 we use the config name rather than structure name to help people figure out what it is thats wrong - they dont know the structure names